### PR TITLE
Logging assert vis

### LINKED ISSUE #84: Add logging and assertions for visualization

### PR DESCRIPTION
**Pull request by wwweert123**
fix #84 
Hi guys please take a look at my pull request!

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM, maybe can add extra spaces between large blocks of code to make it more readable. Good job!

**Review by NeoMinWei — APPROVED** [review]
Very good logs
Even better asserts
Nothing less from the best
LGTM

File: src/main/java/seedu/financialplanner/visualisations/Categorizer.java
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/visualisations/Categorizer.java:1]: good use of logging

**Review by NeoMinWei — COMMENTED** [review]
LGTM

**Pull request merged**